### PR TITLE
Removed unused class imports

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

Just some cleanup to remove "a few" unused class imports.

### Fixed Issues (if relevant)

1. n/a

### Manual testing scenarios (*)

1. n/a

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @sreichel. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: dev/tests/functional/tests/app/Magento/Newsletter/Test/Handler/Template/Curl.php [hunk 1]
```diff
@@ -8,7 +8,6 @@
 
 use Magento\Mtf\Fixture\FixtureInterface;
 use Magento\Mtf\Handler\Curl as AbstractCurl;
-use Magento\Mtf\Util\Protocol\CurlTransport;
```
Comment (sivaschenko) [inline dev/tests/functional/tests/app/Magento/Newsletter/Test/Handler/Template/Curl.php]: CurlTransport is used in this class

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for your contribution @sreichel , I reviewed several randomly selected files and found an issue. Please see the review comment.

**Comment by sivaschenko** [comment]
Thanks for the contribution, @sreichel ! Please see the review notes. Also, it would be great if you can keep the number of files updated by such kind of pull requests lower (maybe splitting between several pull requests), as it's hard to properly review and I experience performance issues on github while viewing the code changes. :)

**Comment by sreichel** [comment]
@sivaschenko OK, ill split it next time :)

For the mentention used class ... it's still there in my code (?)

snip-missing-class

Wait for travis and check it. Thanks.

**Comment by sreichel** [comment]
@sivaschenko I'll split it next time.

I'm on nearly latest M2.3-dev, but there  is no `CurlTransport` class in code.

snip-missing-class

See: https://github.com/magento/magento2/tree/2.3-develop/dev/tests/functional/lib/Magento/Mtf/Util

File: dev/tests/functional/tests/app/Magento/Newsletter/Test/Handler/Template/Curl.php (hunk 1 above)
Comment (sreichel) [reply dev/tests/functional/tests/app/Magento/Newsletter/Test/Handler/Template/Curl.php]: Please see answer below ... this class does not exist.

**Comment by sivaschenko** [comment]
@sreichel you are missing the MTF installation. Please navigate to `dev/tests/functional/` and run `composer install`

**Comment by sreichel** [comment]
@sivaschenko sorry, my bad. Installed MTF and class is there, but ... I don't know how to fix it. Please see file at my M2 fork ... https://github.com/sreichel/magento2/blob/cleanup/array-interface/dev/tests/functional/tests/app/Magento/Newsletter/Test/Handler/Template/Curl.php

I can see `use Magento\Mtf\Util\Protocol\CurlTransport;` so i dont know whats wrong with the PR.

**Comment by sreichel** [comment]
@sivaschenko sorry for confusion. I was on wrong branch ...

**Comment by sreichel** [comment]
@sivaschenko completly missed MTF ... i'll fix tomorrow.

**Review by orlangur — CHANGES_REQUESTED** [review]
Hi @sreichel,

We already faced with similar contribution a bunch of times. It does not make sense without a proper static test implemented similar to `testCodeStyle` which involves `php-cs-fixer`. 

Please implement such test, without it we unfortunately cannot accept such PR.

**Comment by sivaschenko** [comment]
@orlangur it looks like you already faced this question, can you please provide some guidance, you are aware of, on how to create such test (maybe at least entry points where to place it in the code), any suggestions or recommendations. It will be very helpful and I believe it will speed up the contribution. Thanks!

**Comment by orlangur** [comment]
@sivaschenko I already mentioned `testCodeStyle`, it lies in https://github.com/magento/magento2/blob/2.3-develop/dev/tests/static/testsuite/Magento/Test/Php/LiveCodeTest.php.

**Comment by sreichel** [comment]
@orlangur as @sivaschenko  said, some guidance would be helpfull. I've no idea how to include `testCodeStyle` into this PR.

**Comment by orlangur** [comment]
@sreichel `testCodeStyle` is similar test involving `phpcs`. Check how it is implemented and create a similar test involving `php-cs-fixer` (which is able to remove unused imports automatically).

**Comment by orlangur** [comment]
Closing due to inactivity. Feel free to reach me out anytime later if you wish to continue work on this pull request and it will be reopened.
Thanks for collaboration!

**Pull request closed without merging**